ui/lib/core/addon/components/string-list.js

```diff
@@ -37,8 +38,10 @@ export default class StringList extends Component {
     });
     this.type = this.args.type || 'array';
     this.setType();
-    this.toList();
-    this.addInput();
+    next(() => {
+      this.toList();
+      this.addInput();
+    });
```

Comment:
This was throwing an error about modifying twice in the same computation and preventing the view from loading so I used `next` to work around it.